simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListFragmentParameters.kt

```diff
@@ -9,9 +9,9 @@ import java.io.Serializable
 data class AccountListFragmentParameters(
 
   /**
-   * If set to `true`, then show the library registry menu in the toolbar.
+   * If set to `true`, then fragment won't navigate after a new account is added
    */
 
-  val shouldShowLibraryRegistryMenu: Boolean
+  val addMultipleAccounts: Boolean
 
```

Comment:
I'm not sure about what this PR does, but wouldn't be the responsibility of `MainFragment` to decide to navigate or not?

Reply:
To be clear, it looks like it's already up to the `MainFragment` to decide it in the way you've implemented that. However, I introduced a general and flexible navigation API to avoid passing such arguments to Fragments, because it's not their concern at all. If my API happens to fail to fulfil its goal, I'd like to know the reason.